examples/ai_society/task_generation.py

```diff
@@ -55,7 +56,7 @@ def generate_tasks(role_names: str, task_generator_prompt: str,
         file.write("\n".join(tasks))
 
 
-def main() -> None:
+def main(model=None) -> None:
```

Comment:
I explicitly decided to go without the annotation for 'model' since it is an example and an example must be as simple as possible. And managing the parameter for testing is not what a user should focus on.

Reply:
Sure